source/adapters/cuda/program.cpp

```diff
@@ -382,6 +380,7 @@ urProgramGetInfo(ur_program_handle_t hProgram, ur_program_info_t propName,
     UR_ASSERT(getKernelNames(hProgram), UR_RESULT_ERROR_UNSUPPORTED_FEATURE);
     return UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION;
   case UR_PROGRAM_INFO_NUM_KERNELS:
+  case UR_PROGRAM_INFO_IL:
```

Comment:
I'm wondering, if we wanted to implement this for CUDA, we would have to find way to figure out which binaries are PTX right?

Reply:
yeah, so it's doable but we should probably wait for a use case to come up before going to the trouble